hcl2template/types.packer_config.go

```diff
@@ -177,42 +177,29 @@ func (c *PackerConfig) decodeInputVariables(f *hcl.File) hcl.Diagnostics {
 	return diags
 }
 
-// parseLocalVariables looks in the found blocks for 'locals' blocks. It
-// should be called after parsing input variables so that they can be
-// referenced.
-func (c *PackerConfig) parseLocalVariables(f *hcl.File) ([]*LocalBlock, hcl.Diagnostics) {
+// parseLocalVariableBlocks looks in the AST for 'local' and 'locals' blocks and
+// returns them all.
+func parseLocalVariableBlocks(f *hcl.File) ([]*LocalBlock, hcl.Diagnostics) {
```

Comment:
I removed the `PackerConfig`, so that this function is only an extraction function.

Reply:
Thanks for the call out here.